src/main/host/syscall/protected.c

```diff
@@ -45,26 +47,47 @@ void _syscallhandler_setListenTimeoutMillis(SysCallHandler* sys,
     _syscallhandler_setListenTimeout(sys, &timeout, TIMEOUT_RELATIVE);
 }
 
-int _syscallhandler_isListenTimeoutPending(SysCallHandler* sys) {
+static const Timer* _syscallhandler_timeout(const SysCallHandler* sys) {
```

Comment:
I think `_getTimeout` is a better name here - since `timeout` can be used as a verb, it could be mistaken for a setter (a mistake I made).